src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,229 @@
+        }
+        String todoDescription = userInputs[1].strip();
+        taskList[Task.getNumOfTasks()] = new Todo(todoDescription);
+        printAddTask(taskList[Task.getNumOfTasks() - 1]);
+    }
+
+    private static void executeDone(String[] userInputs) {
+        int itemNum = Integer.parseInt(userInputs[1].replaceAll("[^0-9]", ""));
+        taskList[itemNum - 1].setDone();
+        printDoneTask(taskList[itemNum - 1], itemNum);
+    }
+
+    private static String[] readUserInput(Scanner in) {
+        System.out.println("\tCall out a smurf to do a job for you!");
+        String userInput = in.nextLine();
+        return userInput.strip().split(" ", 2);
+    }
+
+    private static void printWelcomeMessage() {
+        System.out.println(logoArt + "\n" + nameArt);
+        System.out.println(SEPARATOR);
+        System.out.println("\t...la la la la la la sing a happy song\n");
+        System.out.println("\tWelcome to Smurf Village.");
+        System.out.println("\tStart smurfing now!!");
+        System.out.println(SEPARATOR);
+    }
+
+    private static void printDoneTask(Task task, int itemNum) {
+        System.out.println(SEPARATOR);
+        System.out.println("\tBrainy Smurf: aah another thing done\n");
+        System.out.printf("\t%d. [%s][%s] %s\n", itemNum, task.getTaskIcon(), task.getStatusIcon(),
+                task.getDescription());
+        System.out.println(SEPARATOR);
+    }
+
+    private static void printAddTask(Task task) {
+        System.out.println(SEPARATOR);
+        System.out.println("\tHandy Smurf is here to give you a hand!\n");
+        System.out.println("\tI have added: ");
+        System.out.println("\t" + task);
+        System.out.println(SEPARATOR);
+    }
+
+    private static void printTaskList(Task[] taskList) {
+        System.out.println(SEPARATOR);
+        System.out.println("\t\"Tracker Smurf!! I need you here!!\"\n");
+        for (int i = 0; i < Task.getNumOfTasks(); i++) {
+            System.out.println("\t" + taskList[i]);
+        }
+        System.out.println(SEPARATOR);
+    }
+
+    private static void printErrorMessage() {
+        System.out.println(SEPARATOR);
+        System.out.println("\toops i can't find a smurf to help you with your task.");
+        System.out.println(SEPARATOR);
+    }
+
+    private static void printEmptyDescriptionErrorMessage() {
+        System.out.println(SEPARATOR);
+        System.out.println("\tHey you needa gimme a description!!");
+        System.out.println(SEPARATOR);
+    }
+
+    private static void printEmptyTimeFieldErrorMessage() {
+        System.out.println(SEPARATOR);
+        System.out.println("\tHey you didn't provide a time :(");
+        System.out.println(SEPARATOR);
+    }
+
+    private static void printExitMessage() {
+        System.out.println(SEPARATOR);
+        System.out.println("\toh shucks! Gargamel is here..we gotta hide");
+        System.out.println(SEPARATOR);
+    }
+}
```

Comment:
The Duke class contains a bit too many different methods. How about separate some of the methods to other classes? For example, you may have a PrintManager class that only manages all the printing methods.